eng/common/scripts/Helpers/AzSdkTool-Helpers.ps1

```diff
@@ -0,0 +1,192 @@
+Set-StrictMode -Version 4
+
+function Get-SystemArchitecture {
+    $unameOutput = uname -m
```

Comment:
Does this work on windows?

Reply:
Probably not. Most of this script is just copied over from the existing test proxy installer script so not sure why it hasn't caused problems.